.submitproject

```diff
@@ -15,5 +15,6 @@
     excludedSubdirectoryPatterns = Source/ThirdParty/skia
     excludedSubdirectoryPatterns = Source/ThirdParty/unifdef
     excludedSubdirectoryPatterns = Source/ThirdParty/xdgmime
+    excludedSubdirectoryPatterns = Source/*/SaferCPPExpectations/
```

Comment:
Can we assume that these will always be one level deep?

Reply:
Yes. And it is unlikely these will be moved since other scripts depend on this path.